docs/source/install.rst

```diff
@@ -0,0 +1,128 @@
+Install
+=======
+
+Installing with ``pip``
+-----------------------
+
+Installing Zipline via ``pip`` is slightly more involved than the average
+Python package.
+
+There are two reasons for the additional complexity:
+
+1. Zipline ships several C extensions that require access to the CPython C API.
+   In order to build the C extensions, ``pip`` needs access to the CPython
+   header files for your Python installation.
+
+2. Zipline depends on `numpy <http://www.numpy.org/>`_, the core library for
+   numerical array computing in Python.  Numpy depends on having the `LAPACK
+   <http://www.netlib.org/lapack>`_ linear algebra routines available.
+
+Because LAPACK and the CPython headers are non-Python dependencies, the correct
+way to install them varies from platform to platform.  If you'd rather use a
+single tool to install Python and non-Python dependencies, or if you're already
+using `Anaconda <http://continuum.io/downloads>`_ as your Python distribution,
+you can skip to the :ref:`Installing with Conda <conda>` section.
+
+Once you've installed the necessary additional depdendencies (see below for
+your particular platform), you should be able to simply run
+
+.. code-block:: bash
+
+   $ pip install zipline
+
+If you use Python for anything other than Zipline, we **strongly** recommend
+that you install in a
+[virtualenv](https://virtualenv.readthedocs.org/en/latest/).  The excellent
+`Hitchhiker's Guide to Python`_ provides an [excellent
+tutorial](http://docs.python-guide.org/en/latest/dev/virtualenvs/) on
+virtualenv.
+
+GNU/Linux
+~~~~~~~~~
+
+On `Debian-derived`_ Linux distributions, you can acquire all the necessary
+binary dependencies from ``apt`` by running:
+
+.. code-block:: bash
+
+   $ sudo apt-get install libatlas-base-dev python-dev gfortran pkg-config
+
+On recent `RHEL-derived`_ derived Linux distributions (e.g. Fedora), the
+following should be sufficient to acquire the necessary additional
+dependencies:
+
+.. code-block:: bash
+
+   $ sudo dnf install atlas-devel gcc-c++ gcc-gfortran libgfortran python-devel redhat-rep-config
+
+On `Arch Linux`_, you can acquire the additional dependencies via ``pacman``:
+
+.. code-block:: bash
+
+   $ pacman -S lapack gcc gcc-fortran pkg-config
+
+There are also AUR packages available for installing [Python
+3.4](https://aur.archlinux.org/packages/python34/) (Arch's default python is
+now 3.5, but Zipline only currently supports 3.4), and
+[ta-lib](https://aur.archlinux.org/packages/ta-lib/), an optional Zipline
```

Comment:
or 2.7. Maybe we should also mention that python2 is available `extra` with pacman.